oci/spec_freebsd.go

```diff
@@ -0,0 +1,40 @@
+package oci
+
+import (
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+func defaultMounts() []specs.Mount {
+	return []specs.Mount{
+                {
+                        Destination: "/proc",
+                        Type:        "procfs",
+                        Source:      "proc",
+                        Options:     []string{"nosuid", "noexec"},
```

Comment:
Any reason for skipping "nodev" from defaults?

Reply:
nodev is not a valid mount option on FreeBSD.

Reply:
> nodev is not a valid mount option on FreeBSD

indeed, and that is because devices can exist only in devfs, so nodev is essentially implicit in all mounts. Some background in https://bugs.freebsd.org/103447.